### ISSUE TITLE
Is it possible to make a website version?

### ISSUE THREAD

**Issue by Wolf-lbh** [TeamNewPipe/NewPipe#7494]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [ x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [ x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?

I want a desktop version so I can have a seemless workflow between my on the go phone viewing and my at home desktop experience.

#### Why do you want this feature?

Basically NewPipe is the best and I love using it on my phone but then sometimes I miss videos when I go to use my Desktop as there is no version of newpipe that works on a desktop short of running it in an emulated android instance. I would really like it if there was a website I could pull up on my desktop. That way I'd be using the same service and it would be easier to keep track of what I'm doing.

**Comment by opusforlife2** [comment]
https://github.com/TeamPiped/Piped

https://piped.kavin.rocks/
